### PR TITLE
Rel 1.55.1

### PR DESCRIPTION
**Pull request by dschep**
## What did you implement

Closes PLAT-1691

draft release: https://github.com/serverless/serverless/releases/edit/untagged-3ce51b75493396c33c83

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** NO

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 1.55.1 (2019-10-23)
```
Comment (pmuens) [inline CHANGELOG.md]: Might be inaccurate once it's released

Comment (medikoo) [reply CHANGELOG.md]: In US Midway it's still 23rd! ;-)

Comment (pmuens) [reply CHANGELOG.md]: > Wait for it...

image

**Comment by pmuens** [comment]
I guess we can merge this since the 1.55.1 release is already live...

**Comment by dschep** [comment]
Yup. thanks :)

File: CHANGELOG.md (hunk 1 above)
Comment (dschep) [reply CHANGELOG.md]: I released it yesterday :)

**Pull request merged**